package.json

```diff
@@ -35,83 +35,71 @@
-    "core-js": "^3.19.1",
-    "css-loader": "^5.2.7",
-    "docz": "^2.3.3-alpha.0",
-    "eslint": "^7.32.0",
-    "eslint-config-airbnb": "^19.0.0",
-    "eslint-loader": "^4.0.2",
-    "eslint-plugin-import": "^2.25.3",
+    "concurrently": "^7.0.0",
+    "copy-webpack-plugin": "^11.0.0",
+    "core-js": "^3.21.1",
+    "css-loader": "^6.7.1",
+    "eslint": "^8.17.0",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-plugin-import": "^2.25.4",
     "eslint-plugin-jsx-a11y": "^6.5.1",
     "eslint-plugin-markdown": "^2.2.1",
     "eslint-plugin-mdx": "^1.16.0",
-    "eslint-plugin-react": "^7.27.0",
+    "eslint-plugin-react": "^7.28.0",
     "eslint-plugin-react-hooks": "^4.3.0",
-    "gatsby-plugin-sass": "^3.2.0",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^27.3.1",
-    "markdownlint-cli2": "~0.3.2",
+    "jest": "^28.1.0",
+    "jest-environment-jsdom": "^28.1.0",
+    "markdownlint-cli2": "~0.4.0",
+    "mini-css-extract-plugin": "^2.6.1",
     "nodemon": "^2.0.15",
-    "postcss": "^8.4.5",
-    "postcss-loader": "^4.3.0",
-    "prism-react-renderer": "^1.2.1",
-    "prop-types": "^15.7.2",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "postcss": "^8.4.6",
+    "postcss-loader": "^7.0.0",
+    "prop-types": "^15.8.1",
     "rimraf": "^3.0.2",
-    "sass": "^1.47.0",
-    "sass-loader": "^10.2.0",
-    "sinon": "^12.0.1",
-    "style-loader": "^2.0.0",
-    "stylelint": "^14.2.0",
-    "stylelint-webpack-plugin": "^2.3.2",
-    "webpack": "^4.46.0",
-    "webpack-cli": "^3.3.12",
-    "webpack-visualizer-plugin": "^0.1.11"
-  },
-  "peerDependencies": {
-    "prop-types": "^15.7.2",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "sass": "^1.49.8",
+    "sass-loader": "^13.0.0",
+    "sinon": "^14.0.0",
+    "style-loader": "^3.3.1",
+    "stylelint": "^14.5.3",
+    "stylelint-webpack-plugin": "^3.1.1",
+    "terser-webpack-plugin": "^5.3.1",
+    "webpack": "^5.69.1",
+    "webpack-cli": "^4.9.2",
+    "webpack-visualizer-plugin2": "^1.0.0"
   },
   "resolutions": {
     "@babel/plugin-transform-spread": "7.12.1"
   },
   "scripts": {
-    "build": "concurrently \"npm run lib:build\" \"npm run docs:build\"",
-    "docs:build": "rimraf .docz docs && docz build --dest docs",
-    "docs:dev": "rimraf .docz && docz dev",
-    "docs:serve": "rimraf .docz && npm run docs:build && docz serve",
-    "lib:build": "concurrently \"webpack --mode development --config webpack.config.js\" \"webpack --mode production --config webpack.config.js\"",
-    "lib:jest": "jest",
-    "lib:test:js": "concurrently \"eslint --ext js,jsx src/lib tests/propTests\" \"npm run lib:jest\"",
-    "lib:test:css": "stylelint \"src/**/*.scss\" --config stylelint.config.js",
-    "markdown:lint": "markdownlint-cli2 \"**/*.md?(x)\" \"#.docz\" \"#dist\" \"#node_modules\" && eslint \"**/*.md?(x)\"",
-    "markdown:watch": "nodemon --watch . --ext md,mdx --exec \"npm run markdown:lint\"",
-    "prepublishOnly": "npm run lib:build",
```

Comment:
`npm run build` might be still necessary as a prepublish task, but I'm not sure.